.github/workflows/audit.yaml

```
@@ -0,0 +1,48 @@
+      - name: (>) Check that ".editorconfig" exists
+        uses: andstor/file-existence-action@v1
+        with:
+          files: '.editorconfig'
+
+      - name: (>) Check "node_modules" is ignored
+        uses: dkershner6/gitignore-parser@v1
+        with:
+          must_deny: 'node_modules'
+
+      - name: (>) Cache "pnpm"
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)